features/works/work_edit_multiple.feature

```diff
@@ -28,22 +28,54 @@ Feature: Edit Multiple Works
       And I should see "Lovely"
 
   Scenario: I can edit multiple works at once
-    Given I am logged in as "author"
-      And I post the work "Glorious" with fandom "SGA"
-      And I post the work "Excellent" with fandom "Star Trek"
-      And I go to my works page
+  Given I am logged in as "author"
+    And I post the work "Glorious" with fandom "SGA"
+    And I post the work "Excellent" with fandom "Star Trek"
+    And I go to my works page
+  When I follow "Edit Works"
+  Then I should see "Edit Multiple Works"
+    And I should see "All"
+    And I should see "None"
+  When I select "Glorious" for editing
+    And I select "Excellent" for editing
+    And I press "Edit"
+  Then I should see "Your edits will be applied to all of the following works"
+    And I should see "Glorious"
+    And I should see "Excellent"
+  When I set the fandom to "Random"
+   And I press "Update All Works"
+  Then I should see "Your edits were put through"
+    And I should see "Random"
+    And I should not see "SGA"
+    And I should not see "Star Trek"
+  When I view the work "Glorious"
+  Then I should see "Random"
+    And I should not see "SGA"
+  When I view the work "Excellent"
+  Then I should see "Random"
+    And I should not see "Star Trek"
+
+  Scenario: I can edit multiple works at once using a pseud
```

Comment:
I don't think this scenario should be included. [The code that this is intended to test](https://github.com/otwcode/otwarchive/blob/b5bf2e7a5cd362d1e350f2be087e8049721a2484/app/controllers/works_controller.rb#L647-L648) can't be accessed through the normal interface, IIRC -- the scenario was only added because the original PR included code to make it accessible. But [the plan is to delete the code](https://otwarchive.atlassian.net/browse/AO3-5476), not make it accessible.

Reply:
Removed